tests/nightwatch/wsManagement.js

```diff
@@ -0,0 +1,29 @@
+describe('Create, rename and remove workspaces', function() {
```

Comment:
Can you split in three describes please ?

- `Create workspaces`
- `Rename workspaces`
- `Remove workspaces`